### PR TITLE
Mayhem Integration

### PR DESCRIPTION
**Pull request by Gediminas19**

### CODE REVIEW

File: mayhem/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,12 @@
+FROM fuzzers/cargo-fuzz:0.10.0
+
+ADD . /pest
+WORKDIR /pest
+RUN rustup toolchain install nightly
+WORKDIR /pest/meta
+RUN cargo bootstrap && RUSTFLAGS="-Znew-llvm-pass-manager=no" cargo +nightly fuzz build
+WORKDIR /pest/grammars
+RUN cargo bootstrap && RUSTFLAGS="-Znew-llvm-pass-manager=no" cargo +nightly fuzz build
+
```
Comment (xansec) [inline mayhem/Dockerfile:10]: Same here. Please copy your resulting binaries into a new package stage to reduce the image size.

**Review by xansec — CHANGES_REQUESTED** [review]

**Comment by xansec** [comment]
Please make the above changes or we will have to close this PR.

**Comment by Gediminas19** [comment]
Sorry, I'll do it tonight

**Comment by xansec** [comment]
LGTM!

**Pull request merged**